docs/operations/clone_api.md

```diff
@@ -0,0 +1,157 @@
+# Clone API
```

Comment:
Is clone aso in Alpha? If so, please add a disclaimer

Reply:
done